community/shellinabox/APKBUILD

```diff
@@ -0,0 +1,61 @@
+# Contributor: Ishak BELAHMAR <ishak@nytrio.com>
+# Maintainer: Ishak BELAHMAR <ishak@nytrio.com>
+pkgname=shellinabox
+pkgver=3.9.6
+pkgrel=0
+pkgdesc="publish command line shell through AJAX interface"
+url="https://github.com/shellinabox/shellinabox"
+arch="x86_64"
+license="GPL-2.0"
+depends="zlib musl"
+makedepends="libressl-dev linux-pam-dev zlib-dev autoconf automake libtool g++"
+install="$pkgname.post-install"
+subpackages="$pkgname-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/shellinabox/shellinabox/archive/master.tar.gz launcher.patch"
+builddir="$srcdir"/shellinabox-master
+
+
+prepare() {
+	patch -p0 -i "$srcdir"/launcher.patch
+}
+
+build() {
+	cd "$builddir"
+	autoreconf -i
+	./configure
+	make
+}
+
+check() {
+	cd "$builddir"
+}
+
+package() {
+	cd "$builddir"
+	make DESTDIR="$pkgdir" install
+
+	# Avoid writing on /usr/local so we fix it here
+	mv "$pkgdir"/usr/local/bin "$pkgdir"/usr/bin
+	mv "$pkgdir"/usr/local/share "$pkgdir"/usr/share
+	rmdir "$pkgdir"/usr/local 
+
+	# Compress man file
+	gzip -c9 "$pkgdir"/usr/share/man/man1/shellinaboxd.1 > "$pkgdir"/usr/share/man/man1/shellinaboxd.1.gz 
+	rm "$pkgdir"/usr/share/man/man1/shellinaboxd.1
+
+	# CSS etc files
+	available="$pkgdir"/etc/shellinabox/options-available
+	enabled="$pkgdir"/etc/shellinabox/options-enabled
+
+	mkdir -p $available $enabled
+
+        install -Dm644 shellinabox/black-on-white.css "${available}/00+Black on White.css"
+        install -Dm644 shellinabox/white-on-black.css "${available}/00_White On Black.css"
+        install -Dm644 shellinabox/color.css          "${available}/01+Color Terminal.css"
+        install -Dm644 shellinabox/monochrome.css     "${available}/01_Monochrome.css"
+        install -Dm644 debian/README.available        "${available}/README"
+        install -Dm644 debian/README.enabled          "${enabled}/README"
+
```

Comment:
All spacing should be done with tab. Please remove the newline after the last install and add one after }